docs/insights/space-level-insights.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Space Level Insights
```

Comment:
Space Level Insights ➡️ Space Insights? Just given that the title for the project one is "Project Insights".

Reply:
No :(